test/metrics.test.js

```diff
@@ -33,9 +33,7 @@ describe("metrics", () => {
     describe("openwhisk", () => {
 
         it("should return nodeVersion metric", () => {
-            assert.deepStrictEqual(Metrics.openwhisk(), {
-                nodeVersion: process.version.substr(1)
-            });
+            assert.strictEqual(Metrics.openwhisk().nodeVersion, process.version.substr(1));
```

Comment:
This change makes sense (more readable), but I wonder how this would fail on Windows?

Reply:
Looks like there is serialization going on, and the comparision fails because of the string quotes which are different...

Reply:
Hmm, this would be weird of `deepStringEqual` to do...

Reply:
It's the only difference I saw... But maybe I missed something else, like spaces, or endline characters.